crates/driver/src/infra/simulator/mod.rs

```diff
@@ -142,17 +142,17 @@ pub enum SimulatorError {
 #[derive(Debug, thiserror::Error)]
 #[error("err: {err:?}, tx: {tx:?}")]
 pub struct WithTxError {
-    err: SimulatorError,
-    tx: eth::Tx,
+    pub err: SimulatorError,
+    pub tx: eth::Tx,
 }
 
 #[derive(Debug, thiserror::Error)]
 pub enum Error {
-    #[error("basic: {0:?}")]
+    #[error("{0:?}")]
```

Comment:
I **believe** this is what `#[error(transparent)]` is for.

Reply:
Not sure if transparent calls for Display but should be the irrelevant in this case.